src/lib/components/SelectField/SelectField.jsx

```diff
@@ -177,15 +191,18 @@ SelectField.propTypes = {
   layout: PropTypes.oneOf(['horizontal', 'vertical']),
   /**
    * Set of options to be chosen from.
+   *
+   * Both option or optgroup are accepted.
    */
-  options: PropTypes.arrayOf(PropTypes.shape({
-    disabled: PropTypes.bool,
-    label: PropTypes.string.isRequired,
-    value: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.number,
-    ]),
-  })).isRequired,
+  options: PropTypes.oneOfType([
```

Comment:
I do not see changes in tests. Please add test to test options with optgroups. And you also need to add section into documentation (README.md) about grouping.

Reply:
@adamkudrna is the documentation ok according to you?

Reply:
Almost! Please see below.